source/documentation/api_key/api_key.md

```diff
@@ -0,0 +1,9 @@
+## API keys
```

Comment:
I think that "API keys" is not the primary topic here. I think this should be "Authentication" (or "Authorisation", I think I prefer the first). As part of this section we can explain how to obtain an API key (or a link to the process, or both) and then how to use it (what you've written here).

Up for discussion but might be a good opportunity to highlight that although we don't currently restrict anything to anonymous requests, we reserve the right to do so in the future e.g. for rate limiting and that we encourage anyone to create one.

Reply:
https://docs.payments.service.gov.uk/#quick-start-guide

Reply:
I agree with @arnau. IMO this content should be the first step to the "quick start section"